vng_api_common/notifications/publish/viewsets.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.core.serializers.json import DjangoJSONEncoder
+from django.utils.module_loading import import_string
+from django.utils.timezone import now
+
+from vng_api_common.models import APICredential
+
+
+class NotificationMixin:
+    kenmerken = None
+    hoofd = None
+
+    def get_nc_url(self):
+        if not hasattr(settings, 'NC_URL'):
+            raise ImproperlyConfigured(
+                "For sending notifications settings should include "
+                "NC_URL parameter"
+            )
+        return settings.NC_URL
```

Comment:
Let's use a prefix here: `NOTIFICATIES_NC_URL`

Reply:
fixed. Renamed to `NOTIFICATIES_URL`